css/amp.css

```diff
@@ -148,6 +148,8 @@ i-amp-sizer {
 
 .-amp-element > [placeholder] {
   display: block;
+  /** Background is important to hide all other placeholders below it. */
+  background-color: #efefef;
```

Comment:
Is this not too big of a change? All existing placeholders will now get the differing background?